### PR TITLE
WLTS-30: FTAs for WLTS-28

### PR DESCRIPTION
**Pull request by CameronGuthrie1**
### JIRA link (if applicable) ###

[WLTS-30](https://tools.hmcts.net/jira/browse/WLTS-30)

### Change description ###

FTAs for WLTS-28

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/functionalTest/resources/features/F-004 - Put Dictionary /F-004.feature [hunk 1]
```diff
@@ -123,5 +123,46 @@ Feature: F-004: Put Dictionary Operation
     And   the response [has the 201 OK status code]
     And   the response has all other details as expected.
     And   a successful call [to verify translations] as in [F-004_Verify],
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```
Comment (sullivangareth) [inline src/functionalTest/resources/features/F-004 - Put Dictionary /F-004.feature:126]: indentation - keep consistent with existing tests

File: src/functionalTest/resources/features/F-004 - Put Dictionary /F-004.feature [hunk 2]
```diff
@@ -123,5 +123,46 @@ Feature: F-004: Put Dictionary Operation
...
+    And   the request [is a new english phrase with a blank welsh translation]
+    And   it is submitted to call the [PUT dictionary] operation of [Translation Service]
+    Then  a positive response is received
+    And   the response [has the 201 OK status code]
+    And   the response has all other details as expected.
+    And   a successful call [to verify translations] as in [F-004_Verify],
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+@S-004.11 #WLTS-28 AC5
+  Scenario: Upload a translation where  translatedPhrase is not supplied for englishPhrase which exists in database
```
[the remark this answers is not in the document]
Comment (CameronGuthrie1) [reply src/functionalTest/resources/features/F-004 - Put Dictionary /F-004.feature:141]: done

File: src/functionalTest/resources/features/F-004 - Put Dictionary /F-004.feature (hunk 1 above)
Comment (CameronGuthrie1) [reply src/functionalTest/resources/features/F-004 - Put Dictionary /F-004.feature:124]: done

**Review by sullivangareth — APPROVED** [review]

**Comment by Thor-tech-of-metal** [comment]
It looks good to me, I have seem it working with WLTS-28 branch . It is working.

**Review by Thor-tech-of-metal — APPROVED** [review]
It looks good to me, I have seem it working with WLTS-28 branch . It is working for the written A.C

**Review by mattnayler — APPROVED** [review]
👍 re-reviewed after repointing to WLTS-28, i.e. #48.

**Pull request merged**